packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:math';
```

Comment:
Change to `import 'dart:math' as math;` as per the style guide: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#import-conventions